www/package.json

```diff
@@ -67,6 +67,7 @@
     "react-icons": "^2.2.7",
     "react-instantsearch": "^4.5.1",
     "react-modal": "^3.4.4",
+    "react-scroll": "^1.7.10",
```

Comment:
https://bundlephobia.com/result?p=react-scroll@1.7.10 this adds 6kB to bundle, I don't think it's worth extra bandwitdh for scrolling animation (which we aren't even using anywhere else)